dev/tests/api-functional/testsuite/Magento/GraphQl/Customer/RequestPasswordResetEmailTest.php

```diff
@@ -0,0 +1,101 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\GraphQl\Customer;
+
+use Magento\TestFramework\TestCase\GraphQlAbstract;
+
+class RequestPasswordResetEmailTest extends GraphQlAbstract
+{
+    /**
+     * @magentoApiDataFixture Magento/Customer/_files/customer.php
+     */
+    public function testCustomerAccountWithEmailAvailable()
+    {
+        $query =
+            <<<QUERY
+mutation {
+  requestPasswordResetEmail(email: "customer@example.com")
+}
+QUERY;
+        $response = $this->graphQlMutation($query);
+
+        self::assertArrayHasKey('requestPasswordResetEmail', $response);
+        self::assertTrue($response['requestPasswordResetEmail']);
+    }
+
+    /**
+     * Check if customer account is not available
+     */
+    public function testCustomerAccountWithEmailNotAvailable()
+    {
+        $query =
+            <<<QUERY
+mutation {
+  requestPasswordResetEmail(email: "customerNotAvalible@example.com")
+}
+QUERY;
+        $this->assertMessage('No such entity with email = customerNotAvalible@example.com, websiteId = 1');
```

Comment:
That's definitely redundant info `websiteId = 1`. Should be resolved with changing exception message.

Reply:
**Done**